Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Viewport/ModularViewportCameraController.h

```diff
@@ -16,37 +16,44 @@
 
 namespace AtomToolsFramework
 {
-    class ModernViewportCameraControllerInstance;
+    class ModularViewportCameraControllerInstance;
+
+    //! Builder class to create and configure a ModularViewportCameraControllerInstance.
     class ModularViewportCameraController
         : public AzFramework::MultiViewportController<
-              ModernViewportCameraControllerInstance, AzFramework::ViewportControllerPriority::DispatchToAllPriorities>
+              ModularViewportCameraControllerInstance,
+              AzFramework::ViewportControllerPriority::DispatchToAllPriorities>
     {
     public:
         using CameraListBuilder = AZStd::function<void(AzFramework::Cameras&)>;
         using CameraPropsBuilder = AZStd::function<void(AzFramework::CameraProps&)>;
 
-        //! Sets the camera list builder callback used to populate new ModernViewportCameraControllerInstances
+        //! Sets the camera list builder callback used to populate new ModularViewportCameraControllerInstances
         void SetCameraListBuilderCallback(const CameraListBuilder& builder);
-        //! Sets the camera props builder callback used to populate new ModernViewportCameraControllerInstances
+        //! Sets the camera props builder callback used to populate new ModularViewportCameraControllerInstances
         void SetCameraPropsBuilderCallback(const CameraPropsBuilder& builder);
         //! Sets up a camera list based on this controller's CameraListBuilderCallback
         void SetupCameras(AzFramework::Cameras& cameras);
         //! Sets up properties shared across all cameras
         void SetupCameraProperies(AzFramework::CameraProps& cameraProps);
 
     private:
-        CameraListBuilder m_cameraListBuilder;
-        CameraPropsBuilder m_cameraPropsBuilder;
+        CameraListBuilder
+            m_cameraListBuilder; //!< Builder to generate a list of CameraInputs to run in the ModularViewportCameraControllerInstance.
```

Comment:
Nit: Do we need a newline here? Can't we just place the comment above or something?

Reply:
Sure I can move the comment to the line above (this formatting is performed by `clang-format` to maintain a line limit of 140 characters)